script/introduction/fitting.jl

```diff
@@ -0,0 +1,121 @@
+
+using Pumas, Plots, Random
+Random.seed!(0);
+
+
+repeated_dose_regimen = DosageRegimen(100, time=0, ii=4, addl=2)
+
+
+choose_covariates() = (Wt = rand(55:80),)
+
+
+pop = Population(map(i -> Subject(id = i,
+                                  evs = repeated_dose_regimen,
+                                  obs=(dv=Float64[],),
+                                  cvs = choose_covariates()),
+                                  1:24))
+
+
+mymodel = @model begin
+  @param   begin
+    cl ∈ RealDomain(lower = 0.0, init = 1.0)
+    tv ∈ RealDomain(lower = 0.0, init = 10.0)
+    ka ∈ RealDomain(lower = 0.0, init = 1.0)
+    q  ∈ RealDomain(lower = 0.0, init = 0.5)
+    Ω  ∈ PDiagDomain(init = [0.9,0.07, 0.05])
+    σ_prop ∈ RealDomain(lower = 0,init = 0.03)
+  end
+
+  @random begin
+    η ~ MvNormal(Ω)
+  end
+
+  @covariates Wt
+
+  @pre begin
+    CL = cl * (Wt/70)^0.75 * exp(η[1])
+    Vc = tv * (Wt/70) * exp(η[2])
+    Ka = ka * exp(η[3])
+    Vp = 30.0
+    Q  = q
+  end
+
+  @dynamics Depots1Central1Periph1
+
+  @derived begin
+      cp := @. 1000*(Central / Vc) # We use := because we don't want simobs to store with variable
+      dv ~ @. Normal(cp, sqrt(cp^2*σ_prop))
```

Comment:
```suggestion
      dv ~ @. Normal(cp, abs(cp)*σ_prop)
```
Alternatively, `Normal(cp, abs(cp)*sqrt(σ²_prop))`. I think we should start being consistent about using `σ` as standard deviation parameter and `σ²` when it's a variance and we might generally prefer standard deviations in the parametrization.